### PR TITLE
Removed the file being required

### PR DESCRIPTION
**Pull request by derikwhittaker**
Simple PR to setup the library to not throw when the file is not provided.

### CODE REVIEW

File: RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs [hunk 1]
```diff
@@ -42,9 +42,9 @@ public static IConfigurationBuilder AddRockLib(this IConfigurationBuilder builde
             if (builder == null) throw new ArgumentNullException(nameof(builder));
             if (string.IsNullOrEmpty(jsonConfigPath)) throw new ArgumentException($"'{nameof(jsonConfigPath)}' must be a non-empty string.", nameof(jsonConfigPath));
 
-            // we want the optional value to be false so that it will throw a runtime exception if the file is not found
-            // if this is set to true no exception is throw and no config values are found/returned.
-            builder = builder.AddJsonFile(jsonConfigPath, optional: false);
+            // we want the optional value to be true so that it will not throw a runtime exception if the file is not found
+            // the Rocklib.config.json is should not be required but just used.
```
Comment (JustinMoss) [inline RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs]: This line doesn't make sense.  Possibly just remove, or reword it?

**Comment by bfriesen** [comment]
Looks good.

File: RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs (hunk 1 above)
Comment (bfriesen) [reply RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs]: Probably just remove the comment if it's gonna get edited.

**Comment by JustinMoss** [comment]
Looks good.

**Pull request merged**